.gitlab/pipeline/build.yml

```diff
@@ -147,10 +147,10 @@ build-linux-stable-cumulus:
   variables:
     # Enable debug assertions since we are running optimized builds for testing
     # but still want to have debug assertions.
-    RUSTFLAGS: "-Cdebug-assertions=y -Dwarnings"
+    # RUSTFLAGS: "-Cdebug-assertions=y -Dwarnings"
```

Comment:
remove this change ?

Reply:
this was needed in order to build chainspec without debug assertions, which are slowing down YAP. reverted the commit